### PR TITLE
Added myself on the website

### PR DESCRIPTION
**Pull request by XTremeRox**
My photo and description

### CODE REVIEW

File: _data/students.json [hunk 1]
```diff
@@ -434,5 +434,15 @@
     },
     "imgName": "Piotr.jpg",
     "name": "Piotr"
+},
+{
+    "social": {
+        "github": "Xtremerox",
+        "twitter": "RamswaroopSoren",
+        "facebook": "ramswaroop.xtreme",
+        "mail": "admin@soren.in",
```
Comment (roonyh) [inline _data/students.json]: Last property of a json object should not have a trailing comma. As per the spec this is wrong syntax.

**Comment by roonyh** [comment]
Hey! It will be great having your image up on our website! Could you please fix the syntax error so that the travis build will pass? Thanks!

**Comment by XTremeRox** [comment]
Done! Thanks for that :)

**Comment by tradzik** [comment]
Hello, 
thank you for your changes, they seems to be ok, however it makes difficult to approve your changes now - they need to be rebased (you can read how to do that here - https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request), so please rebase your pull request.

Thanks ;D

**Pull request closed without merging**